packages/wonder-blocks-clickable/src/components/__tests__/clickable-behavior.test.js

```diff
@@ -93,18 +93,33 @@ describe("ClickableBehavior", () => {
             </ClickableBehavior>,
         );
         const button = screen.getByRole("button");
+        expect(button).not.toHaveTextContent("hovered");
+        expect(button).not.toHaveTextContent("pressed");
+
+        fireEvent.mouseEnter(button, {buttons: 1});
+        expect(button).toHaveTextContent("hovered");
```

Comment:
**suggestion (non-blocking):** This test should also assert that the button is not in the pressed state after the call to `mouseEnter`.